### PR TITLE
Docs for Apache Geode request

### PR DESCRIPTION
**Pull request by vaibhavsood**
Proposing Apache Geode http://geode.apache.org/ for creating a new official image, doc PR

### CODE REVIEW

File: geode/README-short.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Apache Geode (incubating) is an open source, distributed, in-memory database for scale-out apps
```
Comment (metatype) [inline geode/README-short.txt]: Please remove the `(incubating)` reference.  Geode is a top-level project.

File: geode/content.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Please refer https://hub.docker.com/r/apachegeode/geode/
```
Comment (tianon) [inline geode/content.md]: This is going to need to be much more verbose before we can consider merging, btw. :wink:

(Figure this was probably just intended as a temporary placeholder while https://github.com/docker-library/official-images/pull/3685 was evaluated.)

File: geode/license.md [hunk 3]
```diff
@@ -0,0 +1 @@
+https://github.com/apache/geode/blob/develop/LICENSE
```
Comment (tianon) [inline geode/license.md]: This gets embedded directly in the image description as-is -- I'd recommend looking at the `license.md` files of other images for ideas of common formattings for this file.

File: geode/github-repo [hunk 4]
```diff
@@ -0,0 +1 @@
+https://github.com/apache/geode.git
```
Comment (tianon) [inline geode/github-repo]: This should _not_ end in `.git` (again, see other folders for examples).  Also, this is going to result in links that include https://github.com/apache/geode/issues (which are disabled on https://github.com/apache/geode), so we'll likely need at _least_ `geode/issues.md` as well to instruct users where to file issues.

Again, see any of the other folders for examples of how this ought to be formatted:
```console
$ ls */issues.md
aerospike/issues.md    fedora/issues.md          r-base/issues.md
amazonlinux/issues.md  ibmjava/issues.md         ubuntu/issues.md
centos/issues.md       php-zendserver/issues.md  xwiki/issues.md
```

**Comment by yosifkit** [comment]
Closing since corresponding PR was put on hold; https://github.com/docker-library/official-images/pull/3685

**Pull request closed without merging**